R/pkg/R/DataFrame.R

```diff
@@ -1370,14 +1370,22 @@ setMethod("dapplyCollect",
 #' columns with data types integer and string and the mean which is a double.
 #' schema <-  structType(structField("a", "integer"), structField("c", "string"),
 #'   structField("avg", "double"))
-#' df1 <- gapply(
+#' result <- gapply(
```

Comment:
if what is returned is a DataFrame it might help to keep it a variant of "df"
in fact, you might want to add `@return` to document to return value and type